tokio/src/runtime/threadpool/mod.rs

```diff
@@ -259,14 +291,14 @@ impl Runtime {
     /// future panics, or if called within an asynchronous execution context.
     pub fn block_on<F, R, E>(&mut self, future: F) -> Result<R, E>
```

Comment:
The `R` and `E` parameters here are now superfluous, but I left them in since people could be depending on them existing if they're doing something like `block_on<_, (), _>()`